<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDChoixDeMoteur</class>
 <widget class="QDialog" name="BDChoixDeMoteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>82</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Type de moteur: </string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTypeDeMoteur">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>23</number>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBoxTypeDeRendement">
       <property name="maximumSize">
        <size>
         <width>323</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Type de rendement</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="RUnDemi">
          <property name="text">
           <string>Rendement 1/2</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="RTroisQuart">
          <property name="text">
           <string>Rendement 3/4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QRadioButton" name="RQuatreQuart">
          <property name="text">
           <string>Rendement 4/4</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>50</height>
        </size>
       </property>
       <property name="title">
        <string>Unité de puissance à considérer</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="KW">
          <property name="text">
           <string>KW</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="ch">
          <property name="text">
           <string>ch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="btn_Annuler">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_OK">
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
